### ISSUE TITLE
A11y_VscodeWebviewUiToolkit_AllComponents_DataGrid_Reflow: The column headers of data grids are not visible properly at 400% zoom.

### ISSUE THREAD

**Issue by Shivamd1** [microsoft/vscode-webview-ui-toolkit#189]
[“Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:** #A11yMAS;#A11yTCS;#A11ySev2;#SH_VscodeWebviewUiToolkit_Aug2021;#Webview UI Toolkit for Visual Studio Code;#Win32;#FTP;#DesktopApp;#WCAG1.4.10;#Zoom;#AINotInScope;#A11yWCAG2.1;

### Environment Details:
Application Name: Vscode Webview Ui Toolkit
Windows Version: Win10

### Prerequisite:
Set resolution to 1280*1024
Zoom in to 400%

### Repro Steps:

1. Launch VS Code
2. Open WebView UI Toolkit: All Component extension
3. Tab to data grid and observe if all the column headers of data grids are visible properly at 400% zoom.

### Actual:
The column headers of data grid are not visible properly at 400% zoom. The column headers are only visible as C.... .

### Expected:
The column headers of data grid should be properly visible properly at 400% zoom.

### User Impact:
Low vision users will not get the proper information about the data grids.

**Comment by hawkticehurst** [comment]
Hey @Shivamd1, checking in again to see if you have verified that the panel issue has been fixed? Thanks!

**Comment by Shivamd1** [comment]
@hawkticehurst Verified and the issue does not repro closing the bug

**Comment by hawkticehurst** [comment]
Thanks!

**Comment by fsteffi** [comment]
**GithubTags:**#Closed;
